worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java

```diff
@@ -396,7 +402,8 @@ public int size() {
             public boolean contains(Object o) {
                 if (o instanceof BlockVector2) {
                     BlockVector2 cv = (BlockVector2) o;
-                    return cv.getX() >= minX && cv.getX() <= maxX && cv.getZ() >= minZ && cv.getZ() <= maxZ;
+                    return cv.getX() >= minX && cv.getX() <= maxX && cv.getZ() >= minZ
+                        && cv.getZ() <= maxZ;
```

Comment:
same line or break at &&

Reply:
Checkstyle rules for worldedit say that the && needs to be on the line after a line break.